resources/tech/where-ai-meets-code.md

```diff
@@ -0,0 +1,39 @@
+# Where AI Meets Code by Michael Feathers
+
+Resource type: Video
+
+https://www.youtube.com/watch?v=g9m3R0NMJ1Y
```

Comment:
@nicolelynn323 I wound up using the original video instead of the blog post about the video you originally linked to. I figured this is the more defacto link to reference.